source/solvers/cahn_hilliard.cc

```diff
@@ -359,7 +364,7 @@ CahnHilliard<dim>::calculate_phase_statistics()
   std::vector<double> local_phase_order_values(n_q_points);
 
   double integral        = 0;
-  double max_phase_value = std::numeric_limits<double>::min();
+  double max_phase_value = -10.0;
```

Comment:
What's the reason for this specific max phase value?

Reply:
I'll switch it back to std::numeric_limits<double>::min(), it was for debugging